### PR TITLE
IRIS-6473 | popup with input

### PR DESCRIPTION
**Pull request by latata**
@xkxd @leshniak @jsutterfield

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,8 +25,9 @@
     "test": "yarn lint"
   },
   "dependencies": {
+    "@wikia/react-design-system": "Wikia/react-design-system",
     "autoprefixer": "^8.4.1",
-    "design-system": "git://github.com/Wikia/design-system.git#13.8.2",
+    "design-system": "git://github.com/Wikia/design-system.git#035570e38721f57b7ea7cd2b735ba1a041a57110",
```
Comment (latata) [inline package.json]: I used line height sass variable which is not yet released in DS

File: package.json [hunk 2]
```diff
@@ -25,8 +25,9 @@
     "test": "yarn lint"
   },
   "dependencies": {
+    "@wikia/react-design-system": "Wikia/react-design-system",
```
Comment (latata) [inline package.json:28]: I used wds-input styles which are not in DS. Will create a ticket to move them to DS.

File: package.json (hunk 1 above)
Comment (xkxd) [reply package.json]: `14.0.0` contains it

File: src/components/HyperlinkingWrapper.js [hunk 3]
```diff
@@ -1,25 +1,74 @@
...
+        this.remove = this.remove.bind(this);
+
+        this.state = {
+            current: null,
+        };
+    }
+
+    create() {
+        this.setState({
+            current: 'create',
```
Comment (xkxd) [inline src/components/HyperlinkingWrapper.js]: please create const with this value

File: src/components/HyperlinkingWrapper.js [hunk 4]
```diff
@@ -1,25 +1,74 @@
...
+        // TODO
+        const { onClose } = this.props;
+
+        onClose();
+    }
+
     render() {
         const { position } = this.props;
+        const { current } = this.state;
+        const isEdit = current === 'edit';
```
Comment (xkxd) [inline src/components/HyperlinkingWrapper.js]: please move it to consts

File: src/components/PostEditor.js [hunk 5]
```diff
@@ -45,13 +59,18 @@ export default class PostEditor extends Component {
         this.quill.on('selection-change', this.onSelection.bind(this));
     }
 
+    close() {
```
Comment (xkxd) [inline src/components/PostEditor.js]: `onHyperlinkTooltipClose` ?

File: src/index.js [hunk 6]
```diff
@@ -2,6 +2,9 @@ import { h, render } from 'preact';
 
 import PostEditor from './components/PostEditor';
 
+// fixme we shouldn't import it here
+import 'design-system/dist/scss/index.scss';
```
Comment (xkxd) [inline src/index.js]: please remove

**Review by xkxd — APPROVED** [review]

**Pull request merged**